### ISSUE TITLE
(throw "dude") gives false positive in CLJS

### ISSUE THREAD

**Issue by borkdude** [clj-kondo/clj-kondo#2762]
cc @jramosg

**Comment by jramosg** [comment]
I will have look, wasn't fixed here? [https://github.com/clj-kondo/clj-kondo/pull/2750](https://github.com/clj-kondo/clj-kondo/pull/2750)

**Comment by borkdude** [comment]
Hmm, should be right? Nonetheless I'm still seeing this error in `.cljs` files.

```
$ clj -M:clj-kondo/dev --lint - --filename foo.cljs <<< '(throw "dude")'
foo.cljs:1:8: warning: Expected: throwable, received: string.
linting took 36ms, errors: 0, warnings: 1
```
